src/components/navigation/LargeScreenProfileDropdown.tsx

```diff
@@ -0,0 +1,26 @@
+import { useCallback } from 'react';
+
+import { useBosComponents } from '@/hooks/useBosComponents';
+import { useAuthStore } from '@/stores/auth';
+import { useVmStore } from '@/stores/vm';
+
+import { VmComponent } from '../vm/VmComponent';
+import * as S from './styles';
+
+export const LargeScreenProfileDropdown = () => {
+  const components = useBosComponents();
+  const near = useVmStore((store) => store.near);
+  const availableStorage = useAuthStore((store) => store.availableStorage);
+  const logOut = useAuthStore((store) => store.logOut);
+
+  const withdrawTokens = useCallback(async () => {
+    if (!near) return;
+    await near.contract.storage_withdraw({}, undefined, '1');
+  }, [near]);
+
+  return (
+    <S.LargeScreenHeaderActionWrapper>
+      <VmComponent src={components.navigation.profileDropdown} props={{ availableStorage, withdrawTokens, logOut }} />
```

Comment:
Using preview link, I mentioned a very long line with "Withdraw ...". The reason of that is because I removed this line
`availableStorage.div(1000).toFixed(2)`
from `ProfilePropdown` component in order to simplify that component. Let's add this check on our side:

```suggestion
      <VmComponent src={components.navigation.profileDropdown} props={{ availableStorage: availableStorage.gte(10) ? availableStorage.div(1000).toFixed(2) : 0, withdrawTokens, logOut }} />
```

Reply:
This is what I mean:

<img width="369" alt="Знімок екрана 2024-03-20 о 17 50 40" src="https://github.com/near/near-discovery/assets/34593263/61cf3fe3-df20-4b11-b041-e100cbaae4ac">

Reply:
Good catch! I updated both the large and small screen headers with this logic.